messages/ja.json

```
@@ -867,5 +870,113 @@
+      "loading": "設定を読み込み中...",
+      "workDuration": "作業時間",
+      "shortBreakDuration": "短い休憩",
+      "longBreakDuration": "長い休憩",
+      "cyclesBeforeLongBreak": "長い休憩までのサイクル数",
+      "alarmSound": "アラーム音",
+      "volume": "音量",
+      "autoStartBreaks": "休憩を自動開始",
+      "autoStartWork": "作業セッションを自動開始",
+      "minutes": "分",
+      "save": "保存",
+      "saving": "保存中...",
```

Biome (1.9.4)

[error] The key placeholders was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)